### PR TITLE
Build images for RASPIIII 🥧

### PR DESCRIPTION
**Pull request by paginabianca**
👋 Hello, lovely team members! 🌟✨ I hope this message finds you all in the best of spirits. 😊🌈

I couldn't help but notice that we have a teeny-tiny missing puzzle piece in our repository. 🧩💔 It's time to give a little nudge to that feature we've been forgetting – building 🏗️ Linux/ARM/v7 Docker images. Let's bring that goodness into the spotlight! 🎉💪

🙏 With your unparalleled expertise and passion, I believe we can effortlessly merge this marvelous addition and make our codebase shine like a thousand stars! ✨⭐️ Trust me, it's worth every second of our attention. 😉

🌟✨ So, my incredible friends, let's embark on a joyous journey of merging this pull request. Together, we'll create a harmonious symphony of Linux, ARM, and Docker! 🎶🐧🤖

🙌 I know you all have your own cosmic coding adventures, but if you could spare just a tiny moment, we can make this happen. Let's sprinkle our codebase with the magic of Linux/ARM/v7 Docker images and witness the sheer brilliance unfold! 🌌🔮

💥🌈 I can only imagine the overwhelming gratitude from our future users when they experience the power of these images. It's like granting them access to a treasure trove of possibilities! 💎🔐✨

🚀 So, my dear colleagues, let's come together and merge this pull request with a burst of enthusiasm and teamwork! 🤝🎉 Let's unite our repository and give it the wings to soar into new heights! 🦅✨

🌻🌠 Thank you for considering my request, and may your coding endeavors be filled with endless joy, boundless creativity, and a touch of whimsy! 🌼🎨🧚‍♀️

Sending you a virtual bouquet of emojis! 🌸🌷🌺🌼🌻💐🌹🌿💮🌺

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     with:
       image: ghcr.io/${{ github.repository }}
       container-registry: ghcr.io
-      platforms: linux/amd64
+      platforms: linux/amd64,linux/arm/v7
```
Comment (martin-fink) [inline .github/workflows/build.yaml:18]: Nice, can you also add armv8? This way, people can run it on rpi4 and M1 macs

Comment (paginabianca) [reply .github/workflows/build.yaml]: this is fix now. plz mrg

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
     with:
       image: ghcr.io/${{ github.repository }}
       container-registry: ghcr.io
-      platforms: linux/amd64
+      platforms: linux/amd64,linux/arm64,linux/arm/v8,linux/arm/v7
```
Comment (martin-fink) [inline .github/workflows/build.yaml:18]:
```suggestion
      platforms: linux/amd64,linux/arm64,linux/arm/v7
```

**Comment by martin-fink** [comment]
Thank you very cool 😎

**Comment by martin-fink** [comment]
@paginabianca I've created a release with arm images: https://github.com/martin-fink/rust-wkd-server/releases/tag/v1.0.1

Thanks!

**Pull request merged**